### PR TITLE
Contribution guidelines

### PR DESCRIPTION
**Pull request by amacado**
Hey! I drafted some guidelines which hopefully help to clean up this project and restructure it. The main idea is based on the previous work by @iamclaytonray to keep each icon request/submission separated from each other.

Feel free to throw in your opinions, this is just a suggestion :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,30 +1,48 @@
...
+</p>
+<h2>Submitting icon</h2>
+<ul>
+  <li>Create a separated pull request for each icon (no matter how many variations)</li>
+  <li>Include the name of the icon in the pull request f.e. "new icon: html5 (original, plain, line)"</li>
+  <li>optional: Add a image of the new icon(s) to the description of the pull request</li>
+  <li>optional: Reference the issues regarding the new icon</li>
+</ul>
+<h3>Icon formats and naming conventions</h3>
+<p>Each icon comes in different variantions:</p>
```
Comment (Berkmann18) [inline CONTRIBUTING.md:13]:
```suggestion
<p>Each icon comes in different variations:</p>
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,30 +1,48 @@
...
-- original
-- original-wordmark
-- plain
-- plain-wordmark
-- line
-- line-wordmark
-
-This is not mandatory, an icon can only have 1 or 2 versions available. Just keep in mind that the minimum is 1 and the maximum 6 (for now).
-
-The plain and line versions (with or without wordmark) are designed to be available in the final icon font. So they need to stay as simple as possible (one color and ensure that the paths are united before to export to svg). You can use a service like [compressor](https://compressor.io/) or [SVG Editor](https://petercollingridge.appspot.com/svg-editor) in order to optimize the svg file.
```
Comment (Berkmann18) [inline CONTRIBUTING.md:21]: Or [SVOMG](https://svgomg.firebaseapp.com/) or SVGO.

Comment (amacado) [reply CONTRIBUTING.md:21]: Your link to https://svgomg.firebaseapp.com/ throws a 404 not found

Comment (Berkmann18) [reply CONTRIBUTING.md:21]: Really? it works on my hand.

Comment (amacado) [reply CONTRIBUTING.md:21]: When I enter https://svgomg.firebaseapp.com/ i get the following response:
image

What about https://jakearchibald.github.io/svgomg/? is that the page you were looking for?

Comment (Berkmann18) [reply CONTRIBUTING.md:21]: That's bizarre, I found that on DuckDuckGo and that same link works on my hand.

> What about https://jakearchibald.github.io/svgomg/? is that the page you were looking for?

Seems like the same tool so yeah.

**Pull request merged**